pkg/util/utils.go

```diff
@@ -137,7 +137,7 @@ func TimeElapsed(start time.Time, friendly bool) string {
 		elapsed = FriendlyDuration(elapsed)
 	}
 
-	return elapsed.String()
+	return fmt.Sprintf("[%v]", elapsed.String())
```

Comment:
Brackets should not be added here... but where the output is used:
https://github.com/budhrg/minishift/blob/03606c244dcf5030a2e0c2d357d7f0f888fb2c59/pkg/minishift/registration/redhat.go#L100